CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contribution Guidelines
+
+Please help us to make this list better. Your pull requests and suggestions are welcome.
+
+* Make an individual pull request for each suggestion.
+  * If none of the existing categories fit for the tool you are suggesting, you can suggest a new category or suggest to rename an existing one.
+  * You may suggest tools you are developing yourself, or tools developed by others. However, we do not want to include general purpose tools, such as text editors, version control systems, or graphics software.
+* Make individual pull requests for changes.
+* You can either fork this repository and make a pull request with your suggestion/changes, or you can create a new branch in this repository. If you choose to create a new branch, please use the following convention:
+  * for suggestions: `tool/[the tools name]`, e.g. `tool/awesome-analysis-tool`
+  * for changes: `change/[label for you change]`, e.g. `change/category-reordering`
```

Comment:
to avoid too much of a branch chaos...?

Reply:
I guess it depends whether we want to design for large scale now or adopt as we grow.

Reply:
LGTM. I prefer the tool/... and  change/... branching.

Reply:
I think it doesn't hurt to prefix branches (people have to create and name them anyways). If nobody thinks it's a bad idea, I'd merge it like that.